src/common/schema.graphql.ts

```diff
@@ -15,6 +15,7 @@ export default gql`
 		dietaryRestrictions: [DietaryRestriction!]! @column
 		userType: UserType! @column
 		phoneNumber: String @column
+		eventsAttended: [ID!]! @column
```

Comment:
Setting this to be optional would fix some of the type errors. Otherwise you can look at where you need to add an eventsAttended field in objects being created for tests.

Reply:
Set to optional